tasks/cra.sh

```diff
@@ -75,9 +75,6 @@ mv package.json.orig package.json
 # Now that we have packed them, call the global CLI.
 # ******************************************************************************
 
-# If Yarn is installed, clean its cache because it may have cached react-scripts
-yarn cache clean || true
```

Comment:
Yes, we need the cache to work offline, if we clean everything the command won't work :)